### ISSUE TITLE
checkpoint/progress files are not being updated

### ISSUE THREAD

**Issue by gantunesm** [CobayaSampler/cobaya#192]
Hi everyone, 
(I am sorry if it's a repetition but I'm new to cobaya and couldn't find anything related.)

I'm running cobaya with mpi and the .checkpoint, .covmat, and .progress files are created but they don't update over time.  The .txt file is updated and according to my log file, it seems to be running ok. Could this be related to my chain being stuck or it is something else?

 Any idea for a possible reason or where to look for the issue will be highly appreciated.

Thanks, 
Gabriela

**Comment by cmbant** [comment]
Is it producing multiple txt chains files as expected for MPI? Make sure you have Cobaya 3.1 if using the latest Pandas.

**Comment by gantunesm** [comment]
Thanks for your support. 
I was using a previous Cobaya version and the multiple txt files were created as expected for MPI. However, after a few attempts to update everything (Cobaya, conda, pandas, etc) the same error persists but the various .txt files are not being created according to the MPI configuration anymore and there is an additional output file ending with 'input.yaml.locked'. If I run cobaya with the same .yml file on a single node it will work fine, which makes me think that probably the error is the settings/configuration of my job script

**Comment by cmbant** [comment]
Yes, likely, using different mpi to run as linked via mpi4py, as reported by the error message.
________________________________
